src/jit/scopeinfo.cpp

```diff
@@ -947,6 +948,7 @@ void                CodeGen::psiBegProlog()
 
                     newScope->scRegister = true;
                     newScope->u1.scRegNum = (regNumberSmall)regNum;
```

Comment:
Yes. You are right. Reading the code carefully, there were other changes needed as well.